index.js

```diff
@@ -37,8 +39,27 @@ class BabelCompiler {
     }, {});
 
     opts.sourceMap = !!config.sourceMaps;
-    if (!opts.presets) opts.presets = ['latest'];
-    if (opts.presets.length === 0) delete opts.presets;
+
+    // Include 'latest' preset only if:
```

Comment:
Maybe better to leave a link to related issues? Just an reminder why this code shouldn't be removed.